src/Server/ServerStrategy.java

```diff
@@ -1,8 +1,9 @@
 package Server;
 
+import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
 
 public interface ServerStrategy {
-    void handleClient(InputStream inFromClient , OutputStream outToClient);
+    void handleClient(InputStream inFromClient , OutputStream outToClient) throws IOException;
```

Comment:
we can do a try and catch inside the method instead of throwing the exception outside the function

Reply:
Yeah it doesn't matter, we'll change it.